test/README.md

```diff
@@ -65,3 +65,34 @@ Install ringpop-go and build the `testpop` executable:
 Run the tests:
 
 	node ringpop-common/test/it-tests.js $GOPATH/src/github.com/uber/ringpop-go/testpop
+
+# Understanding the integration tests
+
+This is a written summary of a one-hour workshop of digging into the integration tests and asking questions from the people who implemented it. As of now, it might be out of date (we will try to keep it up to date though), but the text below can give a good starting point while trying to wrap your head around.
+
+## Glossary
+
+*Slanted names* mean general concepts, names in `monospace` common variable names.
+
+* *SUT*: Subject Under Test.
+* *testpop*: very basic application using ringpop.
```

Comment:
I think we should add: testpop is commonly used by tickcluster and acts as the SUT for the integration test